editor/components/block-list/multi-controls.js

```diff
@@ -33,9 +38,17 @@ function BlockListMultiControls( { multiSelectedBlockUids, rootUID, isSelecting
 	];
 }
 
-export default connect( ( state ) => {
+export default connect( ( state, ownProps ) => {
+	const { rootUID } = ownProps;
+	const uids = getMultiSelectedBlockUids( state );
+
+	const firstIndex = getBlockIndex( state, first( uids ), rootUID );
```

Comment:
Alternatively, we could use `getFirstMultiSelectedBlockUid` and `getLastMultiSelectedBlockUid`, but this isn't very complex logic, so seems reasonable enough.